mozci/push.py

```diff
@@ -27,6 +29,31 @@
 MAX_DEPTH = 14
 
 
+def get_hgmo(branch, rev):
```

Comment:
The `Push` class already has a `self._hgmo` property. Let's call this `automation_relevance` instead.

Reply:
Done with ba985d11558c5573ad938519273de61dfd5dc941.